recipes/radical.utils/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "radical.utils" %}
+{% set version = "0.47.5" %}
+{% set sha256 = "423c1959c7b387b0a54128a7263d46c6673a247fd4e06de4e728b7c81005d109" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [win]
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
Can you add the dependencies at https://github.com/radical-cybertools/radical.utils/blob/v0.47.5/setup.py#L281-L285 here? Are they optional or required?